apis/keda/v1alpha1/scaledobject_types.go

```diff
@@ -239,13 +239,19 @@ func (so *ScaledObject) IsUsingModifiers() bool {
 
 // getHPAMinReplicas returns MinReplicas based on definition in ScaledObject or default value if not defined
 func (so *ScaledObject) GetHPAMinReplicas() *int32 {
-	if so.Spec.MinReplicaCount != nil && *so.Spec.MinReplicaCount > 0 {
+	if so.Spec.MinReplicaCount != nil {
 		return so.Spec.MinReplicaCount
 	}
 	tmp := defaultHPAMinReplicas
 	return &tmp
 }
 
+// GetDefaultHPAMinReplicas returns defaultHPAMinReplicas
+func (so *ScaledObject) GetDefaultHPAMinReplicas() *int32 {
+	tmp := defaultHPAMinReplicas
+	return &tmp
```

Comment:
I don't think this needs a receiver, the `so` is ignored in the body

Reply:
> Quote reply

make sense